lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -472,6 +472,24 @@ def _create_global_staff_user(self):
         self.global_staff = GlobalStaffFactory.create()  # pylint: disable=attribute-defined-outside-init
         assert self.client.login(username=self.global_staff.username, password=TEST_PASSWORD)
 
+    def _get_urls(self):  # lint-amnesty, pylint: disable=missing-function-docstring
+        self.lms_url = reverse(  # pylint: disable=attribute-defined-outside-init
```

Comment:
Assigning these variables to `self.` means that they become available as attributes on the `BaseViewsTestCase` instance after this function is called.

Is that needed, or could we just use normal local variables, like `return lms_url, mfe_url, preview_url`?